### ISSUE TITLE
Choosing # of epochs

### ISSUE THREAD

**Issue by XPellow** [XPellow/ECE4179-Project#2]

**Comment by DHelmersonMonash** [comment]
For smallest network: Adam, lr=0.001, batch size = 50, 50 epochs. Allows pretty much all networks to converge on a 4 class classification problem, using 80% of training data. ( the other 20% is left as a validation set)
